docs/arcaflow/workflows/expressions.md

```diff
@@ -260,6 +260,7 @@ Functions:
 | `toLower(string)`                                | string       | Returns the input with Unicode letters mapped to their lower case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `toUpper(string)`                                | string       | Returns the input with Unicode letters mapped to their upper case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `splitString(string, string)`                    | list[string] | Returns a list of the substrings which appear between instances of the specified separator; the separator instances are not included in the resulting list elements; adjacent occurrences of separator instances as well as instances appearing at the beginning or ending of the input will produce empty string list elements.<br>&nbsp; Param 1: The string to split.<br>&nbsp; Param 2: The separator.                                                                                     |
+| `readFile(string)`                               | string |  Returns the contents of a file as a UTF-8 character string given a file path string. Relative file paths are resolved from the Arcaflow process working directory. Shell environment variables are not expanded.                                                                                     |
```

Comment:
I'd say I'd prefer a comma in the first sentence. The first part is an independent thought.
>  Returns the contents of a file as a UTF-8 character string, given a file path string.